package.json

```diff
@@ -4,6 +4,8 @@
   "description": "Serverless ESBuild example using Typescript",
   "main": "handler.js",
   "scripts": {
+    "debug-test": "node --inspect=0.0.0.0 ./node_modules/.bin/jest --runInBand",
+    "debug-invoke-local": "MINIFY=false SOURCEMAP=true node --inspect=0.0.0.0 ./node_modules/.bin/serverless invoke local",
```

Comment:
@TheOrangePuff : Yes, we can do that. However, I'm not sure if we should do it though. The problem here is that anytime you run `serverless` command from inside your project, node will use the locally installed package instead of the `global` one as shown below:
```
$ cat ~/.aligent_aliases 
alias node-run='docker run --rm -it -v ~/.aws:/home/node/.aws -v ~/.azure:/home/node/.azure -v ~/.npm:/home/node/.npm -v "$PWD:/app" aligent/serverless:latest'
alias serverless='node-run serverless'                                                                                                                                                                                                                                                                                                                                       
$ serverless --version
Framework Core: 3.22.0
Plugin: 6.2.2
SDK: 4.3.2

$ cd ./serverless-aws-nodejs-service-template
$ rm -rf node_modules  
$ node-run npm ci      
$ serverless --version
Running "serverless" from node_modules
Framework Core: 3.18.2 (local) 3.22.0 (global)
Plugin: 6.2.2
SDK: 4.3.2
```